static/app/views/settings/projectSourceMaps/index.tsx

```diff
@@ -1,18 +1,76 @@
+import {Fragment, useCallback} from 'react';
+import styled from '@emotion/styled';
+
+import SwitchButton from 'sentry/components/switchButton';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
 import type {RouteComponentProps} from 'sentry/types/legacyReactRouter';
+import type {Organization} from 'sentry/types/organization';
 import type {Project} from 'sentry/types/project';
+import {FlexContainer} from 'sentry/utils/discover/styles';
+import useOrganization from 'sentry/utils/useOrganization';
 
 import {ProjectSourceMaps} from './projectSourceMaps';
 import {ProjectSourceMapsArtifacts} from './projectSourceMapsArtifacts';
+import {ProjectSourceMapsUploads} from './projectSourceMapsUploads';
 
 type Props = RouteComponentProps<
-  {orgId: string; projectId: string; bundleId?: string; name?: string},
+  {
+    orgId: string;
+    projectId: string;
+    bundleId?: string;
+    name?: string;
+  },
   {}
 > & {
   children: React.ReactNode;
   project: Project;
 };
 
+const hasSourceMapUploadsView = (org: Organization) =>
+  org.features.includes('new-source-map-uploads-view');
+
+export function ToggleSourceMapUploadsView({router}) {
+  const useSourceMapUploadsView =
+    router.location.query.useSourceMapUploadsView === 'true';
```

Comment:
```suggestion
  const hasSourceMapUploadsView =
    router.location.query.useSourceMapUploadsView === 'true';
```
it is just a suggestion as `use` is normally used to indicate a hook